$NetBSD: patch-ag,v 1.1 2007/01/16 17:13:37 dmcmahill Exp $

Find a c++ compiler for c++ code instead of hardcoding gcc as a c++
compiler.  Reported to ribbonsoft [2007-01-16 dmcmahill].

--- dxflib/configure.in.orig	2005-11-22 06:46:56.000000000 -0500
+++ dxflib/configure.in
@@ -21,6 +21,7 @@ esac
 #
 AC_PROG_CC
 AC_PROG_CPP
+AC_PROG_CXX
 AC_PROG_INSTALL
 AC_CHECK_PROG(FIND, find, find, :)
 AC_CHECK_PROG(MAKEDEPEND, makedepend, makedepend, :)
